Cordova.podspec

```diff
@@ -0,0 +1,32 @@
+Pod::Spec.new do |s|
+  s.name         = "Cordova"
+  s.version      = "5.1.0"
```

Comment:
Would it be possible to read this from the package.json file? We have too many copies of the version numbers in too many places, and our automated tool needs to try to keep track of all of them... if possible, it would be best to read the version from an existing spot that's guaranteed to be accurate.

Reply:
(If not, we probably have to add it to https://github.com/apache/cordova-coho/blob/master/docs/platforms-release-process.md#create-release-branch or better include it in https://github.com/apache/cordova-coho/blob/80f5dee833e095dc68de69434998b2e0f2575595/src/versionutil.js#L108-L109)

Reply:
It can't be read from `package.json` AFAIK, but it can be added to `versionutil.js`.

Reply:
It looks like parsing package.json might be possible, based on what I see in the Podspec for React Native: https://github.com/facebook/react-native/blob/d0d8c8b651af6a2883307c777a1dfb753f046f7c/React.podspec#L7-L10

This would be preferable, since it's one less spot that could get missed during version bumps. The version in package.json **must** be correct, so it's safest to use that.

Reply:
Cool - I'll give it a shot. I'll try to get to these changes and submit them by this weekend.